credsweeper/file_handler/text_content_provider.py

```diff
@@ -1,8 +1,9 @@
+from os import read
 from typing import Dict, List, Optional
 
 from credsweeper.file_handler.analysis_target import AnalysisTarget
 from credsweeper.file_handler.content_provider import ContentProvider
-
+from credsweeper.utils import Util
 
 class TextContentProvider(ContentProvider):
```

Comment:
I think there are should be 2 empty lines between class and import, please check

`PEP 8: E302 expected 2 blank lines, found 1`